frontend/public/components/operator-lifecycle-manager/clusterserviceversion.tsx

```diff
@@ -264,13 +270,45 @@ export const ClusterServiceVersionDetails: React.SFC<ClusterServiceVersionDetail
       </div>
     </div>
     <div className="co-m-pane__body">
-      {/* FIXME(alecmerdler): Need to modify CSV conditions to follow standard conventions */}
       <SectionHeading text="Conditions" />
       <Conditions conditions={_.get(status, 'conditions', []).map(c => ({...c, type: c.phase, status: 'True'}))} />
     </div>
   </React.Fragment>;
 };
 
+export const CSVSubscription: React.FC<CSVSubscriptionProps> = (props) => {
+  type SubscriptionProps = {
+    subscription: {loaded: boolean, data?: SubscriptionKind[]};
```

Comment:
```suggestion
    subscription: FirehoseResult<SubscriptionKind[]>;
```

https://github.com/openshift/console/blob/585dfbf26db35b5d566a20228fbe071de78b9f22/frontend/public/components/utils/index.tsx#L95